copter/source/docs/systemid-mode-operation.rst

```diff
@@ -95,3 +95,142 @@ main loop frequency divided by a sub-sample factor.
 | 0            |    0         | 8             |
 +--------------+--------------+---------------+
 
+Identification of a Multicopter
+===============================
+Model Scope
+-------------------------------
+
+The system and its components, that are included in the mathmatical model to be identified is shown in the following block diagram. It covers the motor mixer, the 
+motors and ESCs as well as the airframe of the copter, the gyroscopes and their lowpass filters. 
+The frequency sweep inputs 10-12 are used for the identification, since they provide a direct input to the regarded system. Because of this, the input signal is not 
```

Comment:
I still don't agree with using these inputs as there will be reduced aircraft response at low frequencies and potentially low coherence.  Users should use inputs 1-3 but then use the motor inputs for the input signal of the frequency response analysis.

Reply:
I actually agree with you on this point and I also conducted sysid flight using axes 1-3. Though, the coherence was always lower compared to the results when using inputs 10-12, that is why I continued using these inputs. Here, the coherence is pretty good if the integrator gains are set to 0. Also, I have not tried inputs 1-3 with the default parameters, but my guess is that the coherence will be even lower since the default are pretty low compared to what autotune or the parameter tuning with the identified models produced

Reply:
> Though, the coherence was always lower compared to the results when using inputs 10-12, that is why I continued using these inputs.

That is interesting.  Ok, make the suggestion based on your experience.  I have not done much identification of multicopters.  Helicopter identification is very different as there is much coupling between axes.  So you either have to use special identification process that accounts for the cross coupling between axes and decouples the frequency responses or use a mode that does not feed back attitude or rate in order to uncouple the inputs.  It is very difficult (and scary) to fly.